guides/default-technology-stack.md

```diff
@@ -41,7 +41,11 @@ DfE Digital [Azure DevOps](https://azure.microsoft.com/en-gb/services/devops/) (
 
 ### Containerisation
 
-DfE Digital uses Docker to isolate and package up application depedencies from infrastructure and environment concerns. All repositories should be added to the [`dfedigital`](https://hub.docker.com/u/dfedigital) Docker Hub organisation. Ask in the [`#developers`](https://ukgovernmentdfe.slack.com/messages/CAGBHB4JV) channel on the DfE Slack to have members added to the DfE Docker Hub organisation and for credentials to use with your CI/CD pipeline.
+DfE Digital uses Docker to isolate and package up application depedencies from infrastructure and environment concerns. All repositories should be added to the [`dfedigital`](https://hub.docker.com/u/dfedigital) Docker Hub organisation.
+
+Ask in the [`#developers`](https://ukgovernmentdfe.slack.com/messages/CAGBHB4JV) channel on the DfE Slack to have members added to the DfE Docker Hub organisation and for credentials to use with your CI/CD pipeline.
+
+Please note, it's preferable for the user using CI/CD to be a generic account set up specifically for your project, rather than a personal account, as this saves problems when users move on from a project.
```

Comment:
I'd also add a note about naming the generic account appropriately, i.e. `teacher-services` and also documenting in the team's runbook / opsmanual.

Reply:
Good call 👍